vdb-bench-assembly/plugins/vdb-bench-dataservice/content/css/styles.less

```diff
@@ -706,3 +706,16 @@ button[id^='dropdownKebabRight_'] {
     position: relative;
     top: -4em;
 }
+
+.dsb-wizard-container .modal-header {
+    padding: 0px 15px;
```

Comment:
The angular component has the header.   There is an 'embed-in-page' property that I've set 'true' but the header is still there.  So yeah appears to be leftover